.travis.yml

```diff
@@ -15,6 +15,9 @@ addons:
 before_install:
   - cat test-requirements.txt >> requirements.txt
   - npm install stylelint@7 -g
+  - pip install nltk
+  - python3 -m nltk.downloader punkt maxent_ne_chunker wordnet
+  - python3 -m nltk.downloader averaged_perceptron_tagger words
```

Comment:
this will do the same thing
```yaml
- python3 -m nltk.downloader punkt maxent_ne_chunker wordnet
                             averaged_perceptron_tagger words
```